Options.py

```diff
@@ -700,6 +700,11 @@ class DeathLink(Toggle):
     display_name = "Death Link"
 
 
+class CustomItemPool(OptionDict):
+    """Customize your item pool. Use with care"""
```

Comment:
A more descriptive doc string please. If/when this gets added to the website this gets used for the tooltip and should be able to explain how the option works without wholly relying on an advanced user guide.